src/Sulu/Bundle/MarkupBundle/Listener/SwiftMailerListener.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+
+namespace Sulu\Bundle\MarkupBundle\Listener;
+
+
+use Sulu\Bundle\MarkupBundle\Markup\MarkupParserInterface;
+use Swift_Events_SendEvent;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+class SwiftMailerListener implements \Swift_Events_SendListener
+{
+
+    /**
+     * @var MarkupParserInterface[]
+     */
+    private $markupParser;
+
+    /**
+     * @var RequestStack
+     */
+    private $requestStack;
+
+    /**
+     * @param MarkupParserInterface[] $markupParser
+     */
+    public function __construct(array $markupParser, RequestStack $requestStack)
+    {
+        $this->markupParser = $markupParser;
+        $this->requestStack = $requestStack;
+    }
+
+
+    /**
+     * @inheritDoc
+     */
+    public function beforeSendPerformed(Swift_Events_SendEvent $event)
+    {
+        $message = $event->getMessage();
+
+        $body = $message->getBody();
+        $format = $message->getBodyContentType();
+
+        if (count($explodedFormat = explode('/', $format)) > 1) {
+            $format = $explodedFormat[1];
+        }
+
+        $locale = $this->requestStack->getCurrentRequest()->getLocale();
```

Comment:
`getCurrentRequest` could be empty (e.g. emails send from cli) we need to fallback here to the default (%kernel.locale%).